### DISCUSSION TITLE
Full Node stuck in Syncing

### DISCUSSION THREAD

**Discussion by mapsdog** [Chia-Network/chia-blockchain#17110] [Support]
I rebuilt my computer and started syncing and after two week it is not moving any further.  Stuck at "Syncing 4,613,112/4,680,350

Connection Status is:  Connected

Windows 11Pro

![image](https://github.com/Chia-Network/chia-blockchain/assets/152420306/d362d00e-2cab-478c-b964-7296f1ce7f5e)

**Comment by BrandtH22** [comment]
Hey @mapsdog , as a note we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia). 

As for the stuck at syncing, you might need to:
1. stop chia
2. delete the peers.dat file at `~\.chia\mainnet\db\peers.dat` (this will force chia to connect to new peers)
3. start chia and let it sync

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
